website/src/app/kb/authenticate/oidc/readme.mdx

```diff
@@ -1,29 +1,115 @@
+import Alert from "@/components/DocsAlert";
 import PlanBadge from "@/components/PlanBadge";
+import SupportOptions from "@/components/SupportOptions";
 
-<PlanBadge plans={["starter", "enterprise"]} />
+<PlanBadge plans={["starter", "enterprise"]}>
 
-# SSO with OpenID Connect (OIDC)
+# SSO with OpenID Connect
 
-Firezone supports authenticating users with a universal OIDC connector. Use this
-connector to enable SSO for any OIDC-capable identity provider.
+</PlanBadge>
 
-For specific instructions for a particular provider, select your provider in the
-list below:
+Firezone supports authenticating users with a universal OIDC connector that
+works with any authentication service offering a standard
+[OIDC](https://en.wikipedia.org/wiki/OpenID) authentication mechanism. Use this
+connector to enable authenticating users and admins to Firezone for any
+OIDC-capable identity provider that supports the `authorization_code` grant.
+
+This connector works great for popular hosted providers like Google Workspace,
+Microsoft Entra ID, and Okta and also for self-hosted ones like
+[Keycloak](https://www.keycloak.org) and [Ory](https://www.ory.sh).
+
+<Alert color="warning">
+  User / group sync is **not supported** with the universal OIDC connector. See
+  the [Google Workspace](/kb/authenticate/google), [Microsoft Entra
+  ID](/kb/authenticate/entra), or [Okta](/kb/authenticate/okta) connectors for
+  automatic user / group sync.
```

Comment:
Just a nitpick: this way we are not repetitive and have an extra keyword for search indexers.
```suggestion
  automatic directory sync.
```